scene/main/viewport.h

```diff
@@ -89,6 +89,13 @@ class Viewport : public Node {
 	GDCLASS(Viewport, Node);
 
 public:
+	enum Scaling3DMode {
+		SCALING_3D_MODE_DISABLED,
```

Comment:
This one is unnecessary to me, as disabled should happen when scale is at 1.0. Also I would add a nearest mode, for the folks who are having fun doing PSX style graphics.

Reply:
I've removed `SCALING_3D_MODE_DISABLED` in favor of `scaling_3d_scale = 0.0`. As far as adding a nearest mode, I feel like that would be better suited for a later PR.

Reply:
Disabled would be a scale of 1.0, not 0.0.

Reply:
`SCALING_3D_MODE_DISABLED` is still in this PR so I guess you might have forgotten to push the changes?

Reply:
You are 100% correct on both points. In the PR I have it disable at 1.0. I just made a typo here and put 0.0. It looks like I didn't push, so I did that on top of rebasing it